packages/graphql/package.json

```diff
@@ -41,7 +41,7 @@
         "@types/faker": "5.1.7",
         "@types/is-uuid": "1.0.0",
         "@types/jest": "26.0.8",
-        "@types/jsonwebtoken": "8.5.0",
+        "@types/jsonwebtoken": "^8.5.4",
```

Comment:
This is the only one that's variable 🤔

Reply:
We're a bit all over the place with this, our actual `dependencies` have it, but our `devDependencies` are locked - these could probably all also be prefixed with `^`!

Reply:
Let's not change anything seeming it works just fine, I fear adding a ^ could introduce unforeseen issues.